lib/sensu-handler.rb

```diff
@@ -110,6 +110,24 @@ def api_request(method, path, &blk)
       res
     end
 
+   def stash_request(method, path, data = '', &blk)
```

Comment:
Looks like data here is expected to be a JSON document as a string? Since we know we are going to send a JSON document, would it be more friendly to users for `data` to accept a hash value?  We can dump to string before setting it as the request body.